packages/odyssey-react/src/components/Button/index.tsx

```diff
@@ -49,29 +51,33 @@ export type ButtonProps = {
  * @example
  * <Button variant="primary" onClick={() => {}}>Button label</Button>
  */
-const Button: FunctionComponent<ButtonProps> = (props) => {
-  const { 
+const Button: FunctionComponent<Props> = (props) => {
+  const {
     children,
     disabled,
     onClick,
-    wide,
     variant = "primary",
+    wide,
+    ...rest
   } = props;
 
   const componentClass = classNames("ods-button", {
     [`is-ods-button-${variant}`]: variant,
     "is-ods-button-full-width": wide
   });
 
+  const omitProps = useOmit(rest);
+
   return (
     <button
+      {...omitProps}
```

Comment:
To make sure I'm understanding this correctly, omitProps allows us to pass the required props from the parent (in the case of Tooltip, the `id` for association) without passing everything down to the child?